oat-sa/tao-core
# Fix/ADF-1789/small issues

### Related to: https://oat-sa.atlassian.net/browse/ADF-1789

### Summary

Fix for 2 small issues:
- when deleting a translation, the language needs to be available again in the dropdown
- the original resource language must not be available for translation

### Details

Update the translation form component.

### How to test

- check out the branch: `git checkout -t origin/fix/ADF-1789/small-issues`
- make sure to have the companion branch checked out and up to date too on the other extensions: `feat/HKD-6/integration`
- make sure to have the Feature Flags activated
```
FEATURE_FLAG_TRANSLATION_ENABLED=true
FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER=true
```
- make sure to have the Item Translator role assigned to your user
- have an item ready for translation
- add a translation and delete it: the language must be available again in the dropdown list
- the language of the main item must not be available for translation
